docs/minishift_start.md

```diff
@@ -27,7 +27,7 @@ minishift start
       --memory int                 Amount of RAM to allocate to the Minishift VM. (default 2048)
       --metrics                    Install metrics (experimental)
   -e, --openshift-env value        Specify key-value pairs of environment variables to set on the OpenShift container. (default [])
-      --openshift-version string   The OpenShift version to run, eg. v1.3.1 (default "v1.3.1")
+      --openshift-version string   The OpenShift version to run, eg. v1.3.1 (default "v1.4.0")
```

Comment:
Right, so it really does not matter, but I agree it is a bit odd. Maybe we should interpolate the the version specified into the Makefile, twice. So it effectively mentions the same version twice. Might be better though!? @budhrg @praveenkumar - WDYT?

 If would be

     "The OpenShift version to run, eg. v1.4.0 (default "v1.4.0")"

And if we change the version number it would be for example:

    The OpenShift version to run, eg. v1.5.0 (default "v1.5.0")